library/common/common/lambda_logger_delegate.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+
+#include "common/common/logger.h"
+#include "common/common/thread.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Logger {
+
+class LambdaDelegate : public SinkDelegate {
+public:
+  using LogCb = std::function<void(absl::string_view)>;
```

Comment:
Having a lambda at this layer isn't necessarily problematic, but it's a layer of indirection we haven't introduced for other callback cases (engine callbacks, request callbacks, PBF, etc.). Why not leverage the bridge struct directly here?

Reply:
I originally wrote this class in upstream and then realized I could write it in Envoy Mobile, so the extra lambdas are an artifact of that. Good catch, will update.

Reply:
I will wait for #1355 to land.

Reply:
updated